### PR TITLE
init ocr3 build lint test

### PR DESCRIPTION
**Pull request by dimkouv**
Add a new workflow that builds lints and tests the new ccip module.

### CODE REVIEW

File: core/services/ocr3/plugins/ccip/commit/plugin.go [hunk 1]
```diff
@@ -146,6 +146,8 @@ func (p *Plugin) Observation(ctx context.Context, outctx ocr3types.OutcomeContex
 		"tokenPrices", len(tokenPrices),
 		"observerInfo", p.cfg.ObserverInfo)
 
+	// tmp
```
Comment (winder) [inline core/services/ocr3/plugins/ccip/commit/plugin.go]: nit: make it harder to ignore
```suggestion
	// TODO: replace temporary code
```

**Review by winder — APPROVED** [review]
LGTM, just one nit

**Review by makramkd — APPROVED** [review]

File: core/services/ocr3/plugins/ccip/commit/plugin.go (hunk 1 above)
Comment (dimkouv) [reply core/services/ocr3/plugins/ccip/commit/plugin.go]: nice, i forgot that

**Review by jasonmci — APPROVED** [review]
approved

**Pull request merged**